index.html

```diff
@@ -989,7 +1058,9 @@ <h3>About</h3><br>
     evt.currentTarget.firstElementChild.className += " w3-red";
   }
   document.getElementById("myLink").click();
-  </script>
   
+  
+  </script>
+<script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAK6z1bgOphFUv2M_6aX-xphZ_m9EnLUrQ&callback=myMap"></script>  

```

Comment:
before merging: this API key should be replaced with your own

Follow instructions here: Read more at: https://www.w3schools.com/graphics/google_maps_basic.asp